.husky/pre-commit.mjs

```diff
@@ -11,10 +12,26 @@ const run = (cmd) => new Promise((resolve, reject) => exec(
 const changeset = await run('git diff --cached --name-only --diff-filter=ACMR');
 const modifiedFiles = changeset.split('\n').filter(Boolean);
 
+/* Optional: Run linting before committing
```

Comment:
remove commented out code

Reply:
I've kept this intentionally, since there have been places where linting is part of CI, and if someone wants to mandate it before commit, they can just uncomment this